Libraries/Utilities/deprecatedCallback.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule deprecatedCallback
+ * @flow
+ */
+
+'use strict';
+
+// @deprecated, will be removed on March 1 2016
```

Comment:
Does this mean we'll break everyone still using callbacks? We could give people a bit more time.